client/me/notification-settings/blogs-settings/header.jsx

```diff
@@ -63,29 +55,33 @@ export default React.createClass( {
 		const [ size, count ] = zip.apply( null, counts ).map( tally );
 
 		if ( count === 0 ) {
-			return this.translate( 'no notifications' );
+			return this.props.translate( 'no notifications' );
 		}
 
 		if ( size === count ) {
-			return this.translate( 'all notifications' );
+			return this.props.translate( 'all notifications' );
 		}
 
-		return this.translate( 'some notifications' );
-	},
+		return this.props.translate( 'some notifications' );
+	};
 
 	render() {
+		const { site } = this.props;
+
 		return (
-			<header key={ this.props.blog.wpcom_url } className="notification-settings-blog-settings-header" onClick={ this.toggleExpanded }>
-				<SiteInfo site={ this.props.blog } indicator={ false }/>
-				<div className="notification-settings-blog-settings-header__legend">
+			<header key={ site.wpcom_url } className="blogs-settings__header" onClick={ this.toggleExpanded }>
+				<SiteInfo site={ site } indicator={ false } />
+				<div className="blogs-settings__header-legend">
 					<em>{ this.getLegend() }</em>
 				</div>
-				{ ! this.props.disableToggle ?
-				<div className="notification-settings-blog-settings-header__expand">
-					<a className={ 'noticon noticon-' + ( this.state.isExpanded ? 'collapse' : 'expand' ) }></a>
-				</div> :
-				null }
+				{ ! this.props.disableToggle
+					? <div className="blogs-settings__header-expand">
```

Comment:
Formatting appears a bit off here. We can wrap the `<div>` in a set of parens, and keep the opening `<div>` on a new line for consistency:

```
? (
	<div className="blogs-settings__header-expand">
		<a className={ 'noticon noticon-' + ( this.state.isExpanded ? 'collapse' : 'expand' ) }></a>
	</div>
)
```